### PR TITLE
doc: css modules guide

### LINKED ISSUE #160: CSS Modules documentation

### PR DESCRIPTION
**Pull request by Wadjetz**
close #160

### CODE REVIEW

**Review by flomonster — APPROVED** [review]
LGTM

File: content/docs/guides/contribute/conventions/front.fr.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ _Rédaction en cours_
 `projects/{nom du projet}/studies/{nom de l'étude}/scenarios/{nom du scenario}`
 
 ### Styles & SCSS
-> ATTENTION : en CSS/React, le scope d'une classe ne dépend pas de l'endroit où le fichier est importé mais est valide pour toute l'application. Si vous importez un fichier `scss` au fin fond d'un composant (ce que nous déconseillons fortement par ailleurs), ses classes seront disponibles pour toute l'application et peuvent donc provoquer des effets de bord.
+> ATTENTION : en CSS/React, le scope d'une classe ne dépend pas de l'endroit où le fichier est importé mais est valide pour toute l'application. Si vous importez un fichier `scss` au fin fond d'un composant (ce que nous déconseillons fortement par ailleurs), ses classes seront disponibles pour toute l'application et peuvent donc provoquer des effets de bord. Vous pouvez utiliser les CSS modules pour eviter les conflits
```
Comment (clarani) [inline content/docs/guides/contribute/conventions/front.fr.md:43]: Missing an accent and a point :) 
```suggestion
> ATTENTION : en CSS/React, le scope d'une classe ne dépend pas de l'endroit où le fichier est importé mais est valide pour toute l'application. Si vous importez un fichier `scss` au fin fond d'un composant (ce que nous déconseillons fortement par ailleurs), ses classes seront disponibles pour toute l'application et peuvent donc provoquer des effets de bord. Vous pouvez utiliser les CSS modules pour éviter les conflits.
```

File: content/docs/guides/contribute/conventions/front.fr.md [hunk 2]
```diff
@@ -61,6 +61,46 @@ Si par exemple nous avons un composant `rollingStockSelector` qui propose une li
...
+
+.title {
+  font-size: 24px;
+  color: #333;
+}
+```
+
+2. **Utilisez les classes dans votre composant React** :
+
+Vite transforme les classes en objets qui contiennent les classes hashées (exemple `_container_h3d8bg`) et les utilisé au moment de la génération du bundle, rendant ainsi les classes uniques.
```
Comment (clarani) [inline content/docs/guides/contribute/conventions/front.fr.md:87]:
```suggestion
Vite transforme les classes en objets qui contiennent les classes hashées (exemple `_container_h3d8bg`) et les utilise au moment de la génération du bundle, rendant ainsi les classes uniques.
```

File: content/docs/guides/contribute/conventions/front.fr.md [hunk 3]
```diff
@@ -61,6 +61,46 @@ Si par exemple nous avons un composant `rollingStockSelector` qui propose une li
...
+export function MyComponent() {
+  return (
+    <div className={styles.container}>
+      <h1 className={styles["title"]}>Mon Titre</h1>
+    </div>
+  );
+};
+```
+
+Pour plus d'information vous pouvez regarder la [documentation](https://vitejs.dev/guide/features.html#css-modules) de vite.js
```
Comment (clarani) [inline content/docs/guides/contribute/conventions/front.fr.md:102]:
```suggestion
Pour plus d'information, vous pouvez regarder la [documentation](https://vitejs.dev/guide/features.html#css-modules) de vite.js
```

**Review by clarani — APPROVED** [review]
Left some comments (typo), but LGTM !

**Review by flomonster — COMMENTED** [review]
I tested the website it seems to work. It would be nice to have a front-maintainer review.

**Review by clarani — APPROVED** [review]
LGTM

**Pull request merged**